docs/development/compiler.rst

```diff
@@ -10,7 +10,7 @@ The SPU compiler aims to provide first class compiler support from the different
 
 `MLIR-HLO <https://github.com/tensorflow/mlir-hlo>`_ MLIR-HLO project connects XLA into MLIR world. 
 
-Having canonical lowerings from different frontend frameworks to the MLIR ecosystem would provide much needed relief to hardware vendors to focus on their unique value rather than implementing yet another frontend for SPU. The goal is to be similar to current hardware vendors adding LLVM target support instead of each one also implementing Clang / a C++ frontend.
+Having canonical lowerings from different frontend frameworks to the MLIR ecosystem would provide much needed relief to hardware vendors to focus on their unique value rather than implementing yet another frontend for SPU. The goal is to add LLVM target support similar to current hardware vendors, rather than every vendor implementing a Clang/C++ frontend.
```

Comment:
For current hardware vendors, they just need to add LLVM target support instead of implementing separate Clang/C++ frontends. MLIR-HLO is achieving similar goal.